torch/nn/functional.py

```diff
@@ -991,13 +991,16 @@ def softsign(input):
 """)
 
 
+@torch._jit_internal.weak_script
 def _get_softmax_dim(name, ndim, stacklevel):
-    warnings.warn("Implicit dimension choice for " + name + " has been deprecated. "
-                  "Change the call to include dim=X as an argument.", stacklevel=stacklevel)
+    # type: (str, int, int) -> int
+    warnings.warn("Implicit dimension choice for {} has been deprecated. "
+                  "Change the call to include dim=X as an argument.".format(name))
```

Comment:
This makes the user experience worse by removing the stack level argument. We should simply ignore it in the JIT instead.